packages/ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ui",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
@Sansossio , the application is not even in beta stage, why are we upgrading the version if no release is ready?

Reply:
This is just a test, I wanted to see if the commit "Resolve#" works